lib/ansible/plugins/inventory/linode.py

```diff
@@ -52,13 +56,16 @@
   - eu-west
 types:
   - g5-standard-2
+compose:
+  ansible_host: private_ip
 '''
 
 import os
+import ipaddress
```

Comment:
This should probably have a documented requirement or you should fall back to using `ansible.module_utils.compat.ipaddress` if there's an ImportError